src/connect-zbt-1/about-the-device/about-connect-zbt-1.md

```diff
@@ -0,0 +1,49 @@
+---
+zendesk:
+  article_id: 30878291112349
+  name: About Home Assistant Connect ZBT-1
+  position: 1
+  labels: connect zbt-1, about the device
+---
+
+[Home Assistant Connect ZBT-1](https://www.home-assistant.io/connectzbt1) is a Zigbee and Thread adapter. It was designed specifically to work with Home Assistant. It connects to your Home Assistant system via USB, allowing it to build either a Zigbee or a Thread network via Home Assistant's built-in Zigbee or Thread integrations.
+
+{% callout "important" %}
+
+Home Assistant Connect ZBT-1 is not a standalone product.
+
+- It requires Home Assistant installed on a smart home hub (such as [Home Assistant Green](https://www.home-assistant.io/green/)).
+- It requires Zigbee or Thread end devices to form a network.
+- **Optional**: [Home Assistant Companion app](https://companion.home-assistant.io/) installed on your smart phone.
+  - Note: the app does not run Home Assistant. It connects to your existing Home Assistant instance, allowing you to interact with it remotely.
+
+{% endcallout %}
+
+A few device characteristics are listed below:
+
+- [Home Assistant Connect ZBT-1](https://www.home-assistant.io/connectzbt1) supports either Zigbee or Thread.
+  - You can choose during setup and also switch later, if needed.
+  - Running Zigbee and Thread at the same time on one adapter is not supported.
+- Home Assistant Connect ZBT-1 was formerly known as Home Assistant SkyConnect.
+  - Despite the new name and look, they have the exact same hardware, capability, and support.
+- You can run 2 different Zigbee solutions in Home Assistant.
+  - You can run an adapter with a deCONZ or Zigbee2MQTT add-on alongside a Home Assistant Connect&nbsp;ZBT-1 with [Zigbee Home Automation](https://www.home-assistant.io/integrations/zha/).
```

Comment:
optional: Could some users misunderstand these two bullet points? Like that the indented bullet point kind of makes it sound like the "two different Zigbee solutions" above are deCONZ and Zigbee2MQTT, even though we mean "ZHA and Zigbee2MQTT".

Would it make sense to slightly adjust this? So something like this maybe?
```suggestion
- You can run 2 different Zigbee solutions in Home Assistant: ZHA or Zigbee2MQTT
  - You can run another adapter with a deCONZ or Zigbee2MQTT add-on alongside a Home Assistant Connect&nbsp;ZBT-1 with [Zigbee Home Automation](https://www.home-assistant.io/integrations/zha/).
```

Reply:
yes, that's much better, thx!